andela/bp-esa-frontend
# #166754738 Add tooltips to icons on cards.

#### What does this PR do?
- This PR adds tooltips to icons on automation cards

#### Description of Task to be completed?
- Add a tooltip to the `onboarding/offboarding` icon
- Add a tooltip to the `more information` icon
- Add a tooltip to the `AIS link` icon

#### How should this be manually tested?
1. Git clone the repository.
2. Checkout to the `ft-add-tooltips-166754738` branch.
5. Start the server `npm run start` _(or the `yarn` equivalent)_.
6. Clone the backend repository, `https://github.com/andela/bp-esa-backend`.
7. Checkout to the backend `develop` branch.
8. Start the backend server `npm run start:dev` _(or the `yarn` equivalent)_.
9. Login to the ESA application
10. Click the card view icon.
11. Hover over the `onboarding/offboarding` icon
A tooltip showing whether it in onboarding or offboarding should appear
12. 11. Hover over the `more information` icon
A tooltip showing `Details` should appear.
13. Click the more information icon
14. Hover over the link to AIS icon.
A tooltip showing `Link to AIS` should appear

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[166754738](https://www.pivotaltracker.com/story/show/166754738)

#### Screenshots (If applicable)
##### Details
<img width="277" alt="Screenshot 2019-06-24 at 12 39 04" src="https://user-images.githubusercontent.com/38077368/60008697-307baa00-967d-11e9-84ac-278cb50a1c1a.png">

##### Onboarding
<img width="287" alt="Screenshot 2019-06-24 at 12 39 30" src="https://user-images.githubusercontent.com/38077368/60008769-51dc9600-967d-11e9-89e4-23d5fa0e3348.png">

##### Offboarding
<img width="280" alt="Screenshot 2019-06-24 at 12 39 22" src="https://user-images.githubusercontent.com/38077368/60008814-628d0c00-967d-11e9-8c50-c5cc3ca5340e.png">

##### Link to AIS
<img width="666" alt="Screenshot 2019-06-24 at 12 39 42" src="https://user-images.githubusercontent.com/38077368/60008861-79cbf980-967d-11e9-8e47-5b23c3517530.png">
